src/code42cli/cmds/search/extraction.py

```diff
@@ -61,12 +63,11 @@ def handle_response(response):
                 events = _get_alert_details(sdk, events)
             except Exception as ex:
                 handlers.handle_error(ex)
-        handlers.TOTAL_EVENTS += len(events)
-        event = None
-        for event in events:
-            output_logger.info(event)
```

Comment:
Did miss a discussion where we decided to no longer output the data using the handlers?

Reply:
Same as above.

Reply:
As we are using `echo_via_pager`, this is no longer required. 

Although this will be needed for `send-to` but we will define a new function for it.